events/2024-11-19-galaxy-at-congresso-brasileiro-de-biologia-evolutiva-.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: event-external
+google_form_id: 1731347547
+title: 'Galaxy at  Congresso Brasileiro de Biologia Evolutiva'
+description: A whole day training event highlighting various aspects of Galaxy functionality.
+external: No page
```

Comment:
@shiltemann not sure this is expected

Reply:
guess we can link here: https://sbbevol.org/